Resolve GCC captured structured binding warning

Building `ex6` with GCC 15.1.1 gives the warning:
```
[...]/src/parapara/include/parapara/parapara.h:1811:80: warning: captured structured bindings are a C++20 extension [-Wc++20-extensions]
 1811 |                         if (f.error!=failure::unrecognized_key) f.ctx.cindex = tokens[1].second;
      |                                                                                ^~~~~~
[...]/src/parapara/include/parapara/parapara.h:1778:25: note: declared here
 1778 |             auto [kind, tokens] = parser_(std::string_view(ctx_.record));
      |                         ^~~~~~
```

--- a/include/parapara/parapara.h
+++ b/include/parapara/parapara.h
@@ -1775,9 +1775,10 @@ struct ini_style_importer {
         while (in_) {
             std::getline(in_, ctx_.record);
             ++ctx_.nr;
-            auto [kind, tokens] = parser_(std::string_view(ctx_.record));
+            ini_record ir = parser_(std::string_view(ctx_.record));
+            auto& tokens = ir.tokens;
 
-            switch (kind) {
+            switch (ir.kind) {
             case ini_record_kind::section:
                 section_ = tokens[0].first;
                 ctx_.cindex = tokens[0].second;

